ocrd_tool.md

```diff
@@ -10,6 +10,84 @@ services](swagger).
 
 To validate a `ocrd-tool.json` file, use `ocrd ocrd-tool /path/to/ocrd-tool.json validate`.
 
+## Standard parameters
+
+There is a number of parameters common to all processors that MUST be supported by processors.
+
+### `dpi`
+
+Custom DPI to assume for pixel density of images.
+
+MUST default to 300.
+
```

Comment:
A _default_ here? If you don't make this optional, then trusting the binary image meta-data density value becomes impossible.

Reply:
Good point. But we need a fallback *somewhere* to activate if
* no param provided
* no annotation in PAGE
* none or garbage EXIF data

Reply:
Indeed. The only place I have seen in the spec for that was your recent addition to mets.md:

> However, since technical metadata about pixel density is so often lost in
conversion or inaccurate, processors should assume **300 ppi** for images with
missing or suspiciously low pixel density metadata.

In practise, this is all just a contract around behaviour in core's OcrdExif (or whatever that will be named after we respect the PAGE-XML overrides).